src-data-processing/process-raw-data.R

```diff
@@ -0,0 +1,26 @@
+## script to filter and save metadata on git lfs
+
+## the full open `metadata.tsv` file will be downloaded, made smaller,
+## and placed into the raw-target-data folder
+
+## some code adapted from : https://tutorials.inbo.be/tutorials/r_large_data_files_handling/
+tsv.name <- "./auxiliary-data/raw-target-data/metadata.tsv"
+tsv.url <- paste("https://data.nextstrain.org/files/ncov/open/metadata.tsv.gz",
+                 sep = "")
+if (!file.exists(tsv.name)) {
+  download.file(tsv.url, destfile = paste0(tsv.name, ".gz"))
+  R.utils::gunzip(paste0(tsv.name, ".gz"))
+}
+
+## read data, with only the necessary columns
+dat <- data.table::fread(tsv.name,
+                         select = c("division", "Nextstrain_clade", "date", "date_submitted", "host"))
+dat <- dat[date_submitted >= as.Date("2022-08-01"),]
```

Comment:
We would want to change the date here to 2022-01-01; we need some time before the first target date if we want to fit a model to that target date.

Reply:
what if instead of filtering on `date_submitted`, we filtered on `date >= as.Date("2022-08-01")-150`. Then, each version of data will be filtered "as of" the `date_submitted` field. If I'm understanding what you're saying, we will never use observations with dates prior to 150 days prior to the earliest model fitting date.

Reply:
That would work! Sorry about the confusion with the dates🫣🫣